orga/translations/en.json

```diff
@@ -924,7 +928,6 @@
         "manage-account": "Manage the account",
         "show-actions": "Show actions"
       },
-      "banner": "Students' status is updated automatically to display eligibility for certification.",
```

Comment:
todo: il faudrait rajouter la clef `certificability-banner` avec `not-managing-student` et `sco-managing-students`